lnbits/templates/base.html

```diff
@@ -84,6 +84,14 @@
     {% endif %}
     <!-- scripts from extensions -->
     {% block scripts %}{% endblock %}
+    <!-- dynamic extensions routes-->
+    <script>
+      {% if dynamic_routes %}
+        window.dynamicRoutes = {{ dynamic_routes | tojson }}
```

Comment:
rendering all known routes upfront would solve the issue that public pages for extensions doent load because the route does not exist yet. now the <lnbits-manage-extension-list> is responible for loading, and that does not exist on a public page for non-logged in users.

with this you also dont need to redirect when adding dynamic routes from <lnbits-manage-extensions-list> after a reload